### ISSUE TITLE
Bad documentation or JSON schema for `no-duplicate-imports` with `allow-namespace-imports`

### ISSUE THREAD

**Issue by cruhl** [palantir/tslint#4750]
### Bug Report

- __TSLint version__: 5.16.0
- __TypeScript version__: 3.2.2
- __Running TSLint via__: Visual Studio

#### TypeScript code being linted

[The documentation](https://palantir.github.io/tslint/rules/no-duplicate-imports) for "Config examples" with `no-duplicate-imports` and `allow-namespace-imports` is incorrect...

```json
{
  "rules": {
    "no-duplicate-imports": [true, {"allow-namespace-imports": true}]
  }
}
```

...should be...

```json
{
  "rules": {
    "no-duplicate-imports": {
      "severity": "error",
      "options": {
        "allow-namespace-imports": true
      }
    }
  }
}
```

**Comment by cruhl** [comment]
Here's what VSCode is saying...

![image](https://user-images.githubusercontent.com/4983357/58642599-463ac100-82c3-11e9-8712-ba9e3d06b28f.png)

![image](https://user-images.githubusercontent.com/4983357/58642632-58b4fa80-82c3-11e9-8ee7-0d27ba57751f.png)

**Comment by cruhl** [comment]
Forgot to mention, this works, but still shows the incorrect schema warning...

```json
{
  "rules": {
    "no-duplicate-imports": {
      "severity": "error",
      "options": {
        "allow-namespace-imports": true
      }
    }
  }
}
```

**Comment by adidahiya** [comment]
We don't maintain the JSON schema, that's on https://github.com/SchemaStore/schemastore

as per your original issue description... both options formats are actually supported. the first one is deprecated, while the second one is preferred. unfortunately the rule examples have not all been updated to reflect that (open to PRs).

**Comment by JoshuaKGoldberg** [comment]
☠️ *TSLint's time has come!* ☠️ 

TSLint is no longer accepting most feature requests per #4534. See **[typescript-eslint.io](https://typescript-eslint.io)** for the new, shiny way to lint your TypeScript code with **ESLint**. ✨ 

It was a pleasure open sourcing with you all!

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈 _(#4534)_  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
